### PR TITLE
ZENKO-620 Fix GC Readiness Probes

### PR DESCRIPTION
**Pull request by tmacro**
also fixes replication status probe

### CODE REVIEW

File: lib/BackbeatConsumer.js [hunk 1]
```diff
@@ -180,7 +181,24 @@ class BackbeatConsumer extends EventEmitter {
...
+        let cs = null;
+        if (this._backlogMetrics) {
+            zk = this._zookeeper &&
+                this._zookeeper.getState().code ===
+                    zookeeper.State.SYNC_CONNECTED.code;
+        } else {
+            zk = true;
+        }
+
+        if (this._bootstrap) {
```
Comment (jonathan-gramain) [inline lib/BackbeatConsumer.js:194]: This variable is only set in test mode. Are you sure there's nothing to check on the consumer in the normal case?

File: lib/BackbeatConsumer.js [hunk 2]
```diff
@@ -180,7 +181,24 @@ class BackbeatConsumer extends EventEmitter {
...
+            cs = this._consumer && this._consumer.isConnected();
+        } else {
+            cs = true;
+        }
+        // this._log.error(JSON.stringify(this._zookeeper.getState()));
```
Comment (jonathan-gramain) [inline lib/BackbeatConsumer.js]: Comments to remove

**Review by dora-korpar — APPROVED** [review]

**Review by jonathan-gramain — APPROVED** [review]

**Comment by tcarmet** [comment]
Your PR is stuck in the queue and it seems like your branch doesn't work with development/8.0.

Could you please try rebasing, and see if the tests succeed?

remove my wait command once everything is good for you!

thanks!

**Comment by tcarmet** [comment]
@bert-e wait

**Pull request merged**